caom2-pkg-server/src/main/java/ca/nrc/cadc/caom2/pkg/TarWriter.java

```diff
@@ -193,6 +195,12 @@ public void write(String path, URL url)
             long contentLength = get.getContentLength();
             Date lastModified = get.getLastModified();
             item.contentMD5 = get.getContentMD5();
+            if (item.contentMD5 == null) {
+                URI digest = get.getDigest();
+                if ((digest != null) && (digest.getScheme().equals("md5"))) {
+                    item.contentMD5 = DigestUtil.base64Decode(digest.getRawSchemeSpecificPart());
```

Comment:
the digest URI is in the normal for  we use already - md5:{hex} -  and doesn't need to be decoded.